gate-verify/custom-stage-orca/src/main/java/com/opsmx/plugin/stage/custom/VerificationTriggerTask.java

```diff
@@ -180,6 +193,23 @@ private TaskResult triggerAnalysis(StageExecution stage, Map<String, Object> out
 		}
 	}
 
+	private static void verifyStageParams(JsonObject parameters, GateModel gateModel) throws IllegalArgumentException {
+		if (!parameters.has(LOG_TEMPLATE)) {
+			throw new IllegalArgumentException("Verification gate requires a Log Template to be configured");
+		}
+		if (!parameters.has(METRIC_TEMPLATE)) {
+			throw new IllegalArgumentException("Verification gate requires a Metric Template to be configured");
+		}
+		if (!parameters.has(CANARYRESULTSCORE) || parameters.get(CANARYRESULTSCORE).getAsString().isEmpty() || parameters.get(CANARYRESULTSCORE) == null ||
```

Comment:
same as my comment below for baselinestarttime and canarystarttime... we are throwign exception while triggerAnalysis() -> getPayloadString() uses default values for these parameters - CANARYRESULTSCORE 90, LIFETIME 6, MINICANARYRESULT 70
@sudhakaropsmx please review and confirm so @emanipravallika can fix this
cc @aman-agrawal

Reply:
@rakeshkrmeena We don't have much knowledge on Verification gate that time, @emanipravallika verified only one case. 
issue reporter knowledge based @emanipravallika fixed and verified. 
she explained to me then i proceed to merge.
know i asked to @emanipravallika to create new ticket for this fix and same validation we have in sapor service there also we have to fix.